karakeep/docker-compose.yml

```
@@ -0,0 +1,47 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: karakeep_web_1
+      APP_PORT: 3000
+      PROXY_AUTH_ADD: "false"
+
+  web:
+    image: ghcr.io/karakeep-app/karakeep:0.24.1@sha256:e72341d7626b7f92ea9b4d5e1a671ae1b8b2f6d85873a5f482e836978b650f99
+    user: 1000:1000
+    restart: on-failure
+    environment:
+      MEILI_ADDR: http://meilisearch:7700
+      BROWSER_WEB_URL: http://chrome:9222

```

Use of the full service name.
```suggestion
      MEILI_ADDR: http://karakeep_meilisearch_1:7700
      BROWSER_WEB_URL: http://karakeep_chrome_1:9222

```